VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -793,6 +816,7 @@ Export-ModuleMember -Function Get-PasswordPolicyDefault
 Function Get-PasswordPolicyConfig {
     Param (
         [Parameter (Mandatory = $false)] [ValidateNotNullOrEmpty()] [String]$reportPath,
+        [Parameter (Mandatory = $true)] [ValidateSet('4.4.0','4.5.1','5.0.0')] [String]$version='5.0.0',
```

Comment:
Are we sure we want to default to 5.0.0 or require the version to be set?

Reply:
Yes, I had discussed with Gary and he insisted, it has to be mandatory parameter with default value being 5.0.0 in place.